x-pack/test_serverless/functional/test_suites/search/custom_role_access.ts

```diff
@@ -59,12 +58,14 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
     it('should have limited navigation menu', async () => {
       await pageObjects.svlCommonPage.assertUserAvatarExists();
       // discover navigation link is present
-      await testSubjects.existOrFail('~nav-item-search_project_nav.kibana.discover');
-      // dashboard and index_management navigation links are hidden
-      await testSubjects.missingOrFail('~nav-item-search_project_nav.kibana.dashboard');
-      await testSubjects.missingOrFail(
-        'nav-item-search_project_nav.content.management:index_management'
-      );
+      await testSubjects.existOrFail('~nav-item-id-discover');
+      // index management, dev tools, dashboards and maps navigation links are hidden
+      await testSubjects.missingOrFail('~nav-item-id-management:index_management');
+      await testSubjects.missingOrFail('~nav-item-id-dev_tools');
+      // Playground should be also hidden, probably a bug
+      // await testSubjects.missingOrFail('~nav-item-id-searchPlayground');
```

Comment:
@elastic/search-kibana 

It seems like application privileges are not properly applied to `searchPlayground`. The role I defined has only access to Discover, but I still see and able to open `Playground` link, later to see error message

<img width="1573" alt="Screenshot 2024-10-14 at 13 34 41" src="https://github.com/user-attachments/assets/8c612732-d1cd-4a42-802c-a8d1442ec9b9">

Similar question is if `Connectors` should be visible by default?

Reply:
Yes, this is know and something that is being worked on currently